.github/workflows/build.yml

```diff
@@ -628,7 +703,7 @@ jobs:
     - build-windows-msi
     - build-macos
     - build-ubuntu
-    - build-ubuntu-ssltests
+    - build-ubuntu-ssltests-openssl
```

Comment:
ISTM we should include the new AWS-LC variant in each of these three places as well.  Being included in `allowed-failures` and `allowed-skips` it can't block a merge, but it will ensure that there's a result before a PR is merged.

Reply:
I see. For `allowed-failures` and `allowed-skips`, I agree. But what about `all-required-green`? It looks like that job is "required" to pass, and will fail if any of its members do not pass.

<img width="659" alt="Screenshot 2025-07-04 at 12 28 10 PM" src="https://github.com/user-attachments/assets/12ac1f1f-0bff-431a-884c-a76c709ac295" />

So, it looks like including `-awslc` there would effectively make it a "required" check. I'm open to this, but @gpshead  [expressed a preference](https://discuss.python.org/t/support-building-ssl-and-hashlib-modules-against-aws-lc/44505/16) for making this test non-required at first, which seems reasonable.

Reply:
Oh, I see! `needs` here does not mean "needs to have succeeded", just "needs to have completed. From GitHub's [docs](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idneeds):

> Use jobs.<job_id>.needs to identify any jobs that must complete successfully before this job will run. It can be a string or array of strings. If a job fails or is skipped, all jobs that need it are skipped unless the jobs use a conditional expression that causes the job to continue.

Your proposal makes sense. I'll add `-awslc` in those 3 places.